addon/mixins/prop-types.js

```diff
@@ -2,11 +2,13 @@
  * The PropTypesMixin definition
  */
 import Ember from 'ember'
-const {Mixin, get, getWithDefault, typeOf} = Ember
+const {Mixin, assign, get, getWithDefault, merge, typeOf} = Ember
 import config from 'ember-get-config'
 
 import PropTypes, {getDef, logger, validators} from '../utils/prop-types'
 
+const objectAssign = Object.assign || assign || merge
```

Comment:
I think they want to call it `assignIt`, see here:

https://github.com/juwara0/ember-prop-types/blob/ccf8844162af270f7d4aa9f07b973f2b76387d24/addon/utils/validators/index.js#L22